index.js

```diff
@@ -117,7 +117,7 @@ function bootstrap(options) {
     }
   });
 
-  if (config.env !== 'test' && config.env !== 'ci') {
+  if (config.env === 'production') {
```

Comment:
I'd be tempted to get rid of this condition completely and use explicit log-level configuration for setting log levels.